### ISSUE TITLE
NFR: Search only has to be available in .com

### ISSUE THREAD

**Issue by jonatan-lledo-netcentric** [hlxsites/vg-macktrucks-com#521]
Search has to be available in the main domain so in macktrucks.com. Any other domain has to have the search hidden.

**Comment by manuel-vara** [comment]
Agreed with @Joesturbo to disable search for other domains other than .com
